tests/test_ml.py

```diff
@@ -122,6 +122,26 @@
 }
 TFLITE_FORMAT_2 = ml.TFLiteFormat.from_dict(TFLITE_FORMAT_JSON_2)
 
+GCS_COREML_NAME = 'mymodel.mlmodel'
+GCS_COREML_URI = 'gs://{0}/{1}'.format(GCS_BUCKET_NAME, GCS_COREML_NAME)
+GCS_COREML_URI_JSON = {'gcsCoremlUri': GCS_COREML_URI}
+GCS_COREML_MODEL_SOURCE = ml.CoreMlGCSModelSource(GCS_COREML_URI)
+COREML_FORMAT_JSON = {
+    'gcsCoremlUri': GCS_COREML_URI,
+    'sizeBytes': '23456789'
+}
+COREML_FORMAT = ml.CoreMlFormat.from_dict(COREML_FORMAT_JSON)
+GCS_COREML_SIGNED_URI = GCS_SIGNED_URI_PATTERN.format(GCS_BUCKET_NAME, GCS_COREML_NAME)
+
+GCS_COREML_URI_2 = 'gs://my_bucket/mymodel2.mlmodel'
+GCS_COREML_URI_JSON_2 = {'gcsCoremlUri': GCS_COREML_URI_2}
+GCS_COREML_MODEL_SOURCE_2 = ml.CoreMlGCSModelSource(GCS_COREML_URI_2)
+COREML_FORMAT_JSON_2 = {
+    'gcsCoremlUri': GCS_COREML_URI_2,
+    'sizeBytes': '3456789'
+}
+COREML_FORMAT_2 = ml.CoreMlFormat.from_dict(COREML_FORMAT_JSON_2)
```

Comment:
Values like these should really be calculated in the test cases that need them. Otherwise, any errors raised by these methods will cause the entire test module to fail, instead of just the affected test cases. I don't mind using such "computed constants" sparingly. But I'd recommend doing an audit for such instances, and working them into the test cases as much as possible. Couple of useful guidelines in that regard:

1. If the value is a string (even a formatted string), it's usually ok to define it as a module-level constant.
2. If the value is a collection (dict, list), where all the members are known ahead of time (e.g. `COREML_FORMAT_JSON_2`), it's usually ok to define it as a module-level constant.
3. If the value is a computed object (e.g. `GCS_COREML_MODEL_SOURCE`, `COREML_FORMAT_2`), prefer computing it in the test case that needs it. If it's used in multiple test cases consider pytest fixtures. Define as a module-level constant only as a last resort.

Reply:
done.